keyboards/yandrstudio/sp60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Y&R-Biu (@jiaxin96)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+#if defined(KEYBOARD_yandrstudio_sp60_rev1)
```

Comment:
This is ... gross.

Should move the keymap folder in the rev1 folder, and then make a copy of it in the rev2 folder, and remove the preprocessor stuff.